src/components/Button/template.pug

```diff
@@ -5,12 +5,13 @@ mixin content
             img(loading="lazy" src="{imageSrc}" alt="{imageAlt}")
     +if('label')
         | { label}
+    slot
```

Comment:
what is this slot for inside the button? What's the use case?

Reply:
@zolotokrylin To place custom formatted text here instead of using the label property which accepts strings only.

Use-case: Vote Locking period button: It needs the `7 <br/> days`  formatted text as the child